sdk/resourceconnector/ci.mgmt.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger: none
+
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/azure.resourcemanager.resourceconnector/ci.mgmt.yml
+    - sdk/azure.resourcemanager.resourceconnector/Azure.ResourceManager.ResourceConnector/
+
+
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/azure.resourcemanager.resourceconnector/ci.mgmt.yml
+    - sdk/azure.resourcemanager.resourceconnector/Azure.ResourceManager.ResourceConnector/
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: azure.resourcemanager.resourceconnector
```

Comment:
ServiceDirectory: resourcemanager

Reply:
Would this be the `resourcemanager` or the provider name here ? `resourceconnector`?

Reply:
You are right, this is `resourceconnector`